core/interfaces/branch.py

```diff
@@ -513,7 +513,7 @@ def get_selected_branches(self):
         branches = set()
         for sel in self.view.sel():
             for name in self.get_selected_branch(sel):
-                if name and not name == current_branch_name:
+                if name and not name[1] == current_branch_name:
```

Comment:
Should we check that name[0] is None since I might want to merge remotea/master in to master